docs/examples/demo-select2-with-bootstrap.html

```diff
@@ -40,7 +40,7 @@
 
           <ui-select ng-model="ctrl.person.selected" theme="select2" class="form-control" title="Choose a person">
             <ui-select-match placeholder="Select or search a person in the list...">{{$select.selected.name}}</ui-select-match>
-            <ui-select-choices group-by="'group'" repeat="item in ctrl.people | filter: $select.search">
+            <ui-select-choices group-by="'age'" repeat="item in ctrl.people | filter: $select.search">
```

Comment:
@bobisch My bad, I not sure where I got that from. I think `country` might be better in this case though. Thanks!